### DISCUSSION TITLE
Issue with a small number of QSOs in a logbook

### DISCUSSION THREAD

**Discussion by T012m3n7oR** [magicbug/Cloudlog#1735] [Q&A] [answered]
If you have less than a specific number of QSOs logged, on the Logbook page, you cannot get to the bottom of the "gear" menu for the QSO to delete it.  In my case this number was 5, I'm not sure if this is screen specific or not.  The scroll bar on the right side just scrolls to the bottom of the X QSOs, not the menu.

![image](https://user-images.githubusercontent.com/48076977/199029086-99f5663a-7595-4ebe-8dfb-c6c6f3a0d9c4.png)

**Comment by magicbug** [comment, marked as answer]
This is fixed, just hasn't made it from the dev branch to the master

**Comment by AndreasK79** [comment]
Nope, this was only fixed in search, not in logbook.

**Comment by AndreasK79** [comment]
Just pushed a fix so that the code works everywhere for the dropdown menu,
